scripts/dendrite_sytest.sh

```diff
@@ -52,7 +52,7 @@ fi
 
 # Check for new tests to be added to the test whitelist
 /src/show-expected-fail-tests.sh /logs/results.tap /src/sytest-whitelist \
-    /src/sytest-blacklist || TEST_STATUS=$?
+    /src/sytest-blacklist > /src/test-whitelist.log || TEST_STATUS=$?
```

Comment:
could this file have a name which better represents what it does?

Also: I'd recommend against writing to `/src`. It means that we'll be writing stuff to somebody's working copy of dendrite, and means we can't run it against a read-only mount of the dendrite source.